internal/generator/fluentd/pipeline_to_output.go

```diff
@@ -39,6 +39,17 @@ const (
   </parse>
 </filter>
 {{end}}`
+
+	LogLevelSetterTemplate = `
+# {{.Desc}}
+<filter **>
+    @type record_modifier
+    <record>
+       _dummy_ ${struct_level = record.dig('structured','level'); level = record['level']; if ![nil, level].include?(struct_level) && [nil, 'unknown'].include?(level); record['level']=struct_level; end; nil}
```

Comment:
the default level we decided to use since GCP is `default`

Reply:
`unknown` -> `default` ?

Reply:
yes